core/src/main/java/org/keycloak/representations/idm/OrganizationDomainRepresentation.java

```diff
@@ -58,14 +60,14 @@ public boolean equals(Object o) {
         if (!(o instanceof OrganizationDomainRepresentation)) return false;
 
         OrganizationDomainRepresentation that = (OrganizationDomainRepresentation) o;
-        return name != null && name.equals(that.getName());
+        return name != null && name.equals(that.getName()) && verified == that.isVerified();
```

Comment:
Is `verified` something that differentiate a domain from another or just the name?

Reply:
I'm reverting this change